graylog2-server/src/main/java/org/graylog2/indexer/indexset/IndexSetService.java

```diff
@@ -38,6 +38,13 @@
     Optional<IndexSetConfig> get(String id);
 
     /**
+     * Retrieve the default index set.
+     *
+     * @return A filled {@link Optional} with the default index set, an empty {@link Optional} if there is no default.
```

Comment:
Is there a situation where there is *no* default index set, which is not a serious bug?